test/extensions/common/aws/sigv4a_signer_impl_test.cc

```diff
@@ -501,6 +503,47 @@ TEST_F(SigV4ASignerImplTest, QueryStringDefault5s) {
   EXPECT_TRUE(absl::StrContains(headers.getPathValue(), "X-Amz-Expires=5&"));
 }
 
+TEST(SigV4AKeyDerivationTest, TestKeyDerivations) {
+  auto ec_key =
+      SigV4AKeyDerivation::derivePrivateKey(absl::string_view("akid"), absl::string_view("secret"));
+
+  auto ec_private_key = EC_KEY_get0_private_key(ec_key);
```

Comment:
Please add a 1-liner comment before each test, describing its purpose.
(like the last one, but starting with a capital letter and ending with '.' :) )

Reply:
have added a comment at the top of this test block